src/snadra/_config/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+Priority (high to low)
+    - command line arguments
+    - enviorment variables
+    - config file # TODO
+    - defaults # TODO
+"""
+
+import getopt
+import os
+import sys
+import typing
+
+config: typing.Dict[str, typing.Optional[str]] = {
+    "postgres_host": None,
+    "postgres_user": None,
+}
+
+
+def setupConfig():
```

Comment:
```suggestion
def setup_config():
```

We are not coding in java:)